blocks/productlisting/productlisting.css

```diff
@@ -0,0 +1,140 @@
+body.product-listing-theme {
+  background-color: #f6f6f6;
+  font-family: GothamGD, arial, sans-serif;
+}
+
+body.product-listing-theme main {
+  margin-top: 70;
+}
+
+body.product-listing-theme main .productlisting-wrapper {
+  flex-direction: column;
+  display: grid;
+  grid-template-columns: 1fr min(1440px, 100%) 1fr;
+}
+
+.productlisting {
+  display: flex;
+  grid-column: 2;
+  margin-bottom: 80px;
+  margin-top: 15px;
+  justify-content: flex-start;
+  align-items: stretch;
+  background: #fff;
+  position: relative;
+}
+
+.productlisting div {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+h1 {
```

Comment:
This will apply to all h1 ? Not sure if this is what you want ? Maybe you can be more specific with the selector.